src/test/schema_test.cpp

```diff
@@ -1318,3 +1318,198 @@ TEST(SchemaTest, CustomNodeConfigParamsInvalidType) {
     auto result = ovms::validateJsonAgainstSchema(customNodeConfigParamsInvalidTypeParsed, ovms::MODELS_CONFIG_SCHEMA);
     EXPECT_EQ(result, ovms::StatusCode::JSON_INVALID);
 }
+
+static const char* demultiplexerConfigDemultiplyCount = R"(
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "dummy",
+                "base_path": "dummy_path",
+                "target_device": "CPU",
+                "model_version_policy": {"all": {}},
+                "nireq": 1
+            }
+        }
+    ],
+    "custom_node_library_config_list": [
+        {
+            "name": "dummy_library",
+            "base_path": "dummy_path"
+        }
+    ],
+    "pipeline_config_list": [
+        {
+            "name": "pipeline1Dummy",
+            "inputs": ["custom_dummy_input"],
+            "nodes": [
+                {
+                    "name": "dummyNode",
+                    "library_name": "dummy_library",
+                    "type": "custom",
+                    "params": {
+                        "a": "1024",
+                        "b": "512"
+                    },
+                    "inputs": [
+                        {"b": {"node_name": "request",
+                            "data_item": "custom_dummy_input"}}
+                    ],
+                    "outputs": [
+                        {"data_item": "a",
+                        "alias": "new_dummy_output"}
+                    ],
+                    "demultiply_count": 10
+                }
+            ],
+            "outputs": [
+                {"custom_dummy_output": {"node_name": "dummyNode",
+                                        "data_item": "new_dummy_output"}
+                }
+            ]
+        }
+    ]
+})";
+
+TEST(SchemaTest, DemultiplexerConfigMatchingSchema) {
+    rapidjson::Document demultiplexerConfigMatchingSchemaParsed;
+    demultiplexerConfigMatchingSchemaParsed.Parse(demultiplexerConfigDemultiplyCount);
+    auto result = ovms::validateJsonAgainstSchema(demultiplexerConfigMatchingSchemaParsed, ovms::MODELS_CONFIG_SCHEMA);
+    EXPECT_EQ(result, ovms::StatusCode::OK);
+}
+
+TEST(SchemaTest, DemultiplexerConfigDemultiplyCountNegativeNotAllowed) {
+    rapidjson::Document demultiplexerConfigDemultiplyCountNegativeParsed;
+    demultiplexerConfigDemultiplyCountNegativeParsed.Parse(demultiplexerConfigDemultiplyCount);
+    auto pipeline_list = demultiplexerConfigDemultiplyCountNegativeParsed.FindMember("pipeline_config_list")->value.GetArray();
+    auto node_list = pipeline_list.Begin()->FindMember("nodes")->value.GetArray();
+    node_list.Begin()->FindMember("demultiply_count")->value.SetInt(-1);
```

Comment:
You could do this simpler - like here:
https://github.com/openvinotoolkit/model_server/blob/main/src/test/ensemble_config_change_stress.cpp#L338

Reply:
Then you need 1 config string for most of the tests with setting different values for demultiply count.